packaging/windows/nsis/README.md

```diff
@@ -27,13 +27,19 @@ repository.
 
 ## How to build the installer manually
 1) Install NSIS and make sure `makensis` it is available in the `%PATH%`.
-2) Download and decompress the package assets to your machine.
-3) [Build Khiops in Release mode](https://github.com/KhiopsML/khiops/wiki/Building-Khiops)
+2) Download and decompress the package assets to your machine, on `packaging\windows\nsis\assets`
+3) [Build Khiops in Release mode](https://github.com/KhiopsML/khiops/wiki/Building-Khiops), to build the binaries and the jars
 4) In a console, go to the `packaging/windows/nsis` directory and execute
```

Comment:
I would also add somewhere that we need to position ourselves on the pending release tag (e.g. 10.2.3) before launching the package building process.

Reply:
Done